public/docs/_examples/forms/dart/lib/hero.dart

```diff
@@ -0,0 +1,19 @@
+library hero_form.hero;
+
+class Hero {
+  num number;
+  String name;
+  String power;
+  String alterEgo;
+
+  Hero(this.number, this.name, this.power, {this.alterEgo});
+
+  Map toJson() {
```

Comment:
Could this be just `=> JSON.encode(this)`? See the [library tour's dart:convert coverage](https://www.dartlang.org/docs/dart-up-and-running/ch03.html#dartconvert---decoding-and-encoding-json-utf-8-and-more) for details.